### PR TITLE
Support link preview

### PR DESCRIPTION
**Pull request by shirser121**
Add support for minimal link preview - the current preview MD support

### CODE REVIEW

**Comment by wallisonfelipe** [comment]
this is really usefull

**Comment by whentheis** [comment]
@pedroslopez @PurpShell are you able to merge this, please?

**Comment by shirser121** [comment]
@tuyuribr what should we test?

**Comment by PurpShell** [comment]
> @tuyuribr what should we test?

different URLs/layouts on different devices

**Comment by rizki5411** [comment]
this not work for me

File: src/util/Injected.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ exports.ExposeStore = (moduleRaidStr) => {
     window.Store.Validators = window.mR.findModule('findLinks')[0];
     window.Store.VCard = window.mR.findModule('vcardFromContactModel')[0];
     window.Store.Wap = window.mR.findModule('queryLinkPreview')[0].default;
+    window.Store.queryLinkPreview = window.mR.findModule(x=>x.default && x.default.cache && x.default.name === 'n')[0].default;
```
Comment (tuyuribr) [inline src/util/Injected.js:43]: I think this is an unreliable way to find this module, can we do it in another way?

Comment (PurpShell) [reply src/util/Injected.js:43]: I tend to agree, usually we find modules through common method names within the same module, so for example I used deprecatedSendIq to use deprecatedCastStanza. @shirser121 please figure this one out

**Comment by ahmedalmulki** [comment]
link preview not working for me also .

**Comment by bhavyagosai** [comment]
link preview not working for me too? Is there any work going around this? Need this for a major activity and need YouTube links to show previews in messages

**Comment by StefanoVe** [comment]
any update on this pr? link previews are not currently working for me

**Comment by PurpShell** [comment]
Module invalid

**Pull request closed without merging**